components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -2954,6 +2966,11 @@ private void parseOpenIDConnectConfig(OMElement oauthConfigElem) {
                     requestObjectEnabled = false;
                 }
             }
+            OMElement oAuthAuthzRequest = openIDConnectConfigElem.getFirstChildWithName(getQNameWithIdentityNS
+                    (ConfigElements.OAUTH_AUTHZ_REQUEST));
+            if (oAuthAuthzRequest != null) {
+                defaultOAuthAuthzRequestClassName = oAuthAuthzRequest.getText().trim();
```

Comment:
The variable name is a bit misleading as it is returning the configured and not the default.

Reply:
Fixed with 79bab73a362a2021f071c540d7aed7a385ff3817